nym-vpn-app/src/main.tsx

```diff
@@ -83,6 +84,8 @@ dayjs.extend(duration);
     vpnMode: (await kvGet<VpnMode>('vpn-mode')) || defaultVpnMode,
     uiTheme: await getTheme(),
     welcomeChecked: (await kvGet<boolean>('welcome-screen-seen')) || false,
+    streamingOptimizedLabelSeen:
+      (await kvGet<boolean>(STREAMING_OPTIMIZED_LABEL_FEATURE_KEY)) || false,
```

Comment:
Please see my comment on the `InitState` and why you should not init this boolean here.